canonical-web-and-design/ubuntu.com
# Update what is kubernetes

## Done

- Updated page as per [copy doc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-10728.demos.haus/kubernetes/what-is-kubernetes
- Check against copy doc and see that all changes have been made


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4551
